src/scripts/generate-unary-ops.py

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/env python
+import sys
+import os
+import argparse
+import settings
+import collections
+import random
+import subprocess
+path = os.path.join(settings.TEMPLATE_DIR)
+sys.path.append(path)
+import validate
+
+GENERATED_FILENAME = 'unary-ops-'
+UNARY_OPERATORS = ["+", "-", "++", "--", "!", "~", "typeof", "delete", "void"]
+DEFAULT_TEST_COUNT = 100
+OPERAND_COUNT_MIN = 2
+OPERAND_COUNT_MAX = 11
+DEFAULT_SEED = 10000
+MAX = (2**53) - 1
+MIN = -MAX
+FALSE_VALUES_COUNT = 5
+BITWISE_NOT_MAX = (2**31) - 1
+BITWISE_NOT_MIN = -BITWISE_NOT_MAX
+TEST_CASE_IN_A_FILE = 1000
+TEST_RUNNER_SCRIPT = os.path.join(settings.SCRIPTS_DIR, 'runscripts.sh')
+
+RELATION_OPERATORS = ["<", ">", "<=", ">=", "==", "!="]
+
+def get_arguments():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--op", help = "give the operators to be tested", nargs = '+',
+    default = UNARY_OPERATORS, choices = UNARY_OPERATORS)
+    parser.add_argument("--test-count", help = "give the number of tests to run", type = int, default = DEFAULT_TEST_COUNT)
+    parser.add_argument("--output", help = "give the path to generate files into", default = settings.NUMBER_DIR,)
+    parser.add_argument("--operand-count", help = "give the number of maximum operands in an expression",
+        type = int, choices=range(OPERAND_COUNT_MIN, OPERAND_COUNT_MAX), default = OPERAND_COUNT_MAX)
+    parser.add_argument("--seed", help = "give the random seed value", type = int, default = DEFAULT_SEED)
+    parser.add_argument("--jerry-location", help = "give the location of jerry", required = True)
+    args = parser.parse_args()
+    print(args)
+
+    if len(sys.argv) == 1:
+        parser.print_help()
+        sys.exit(1)
+
+    script_args = parser.parse_args()
+
+    return script_args
+
+def generate_unary_ops_tests(options):
+    random.seed(options.seed)
+    file_count = 1
+    if not os.path.isdir(options.output):
+        os.makedirs(options.output)
+    file_to_open = os.path.join(options.output, '%s%d.js' % (GENERATED_FILENAME, file_count))
+    test_case_counter = 0
+    new_file_begin = True
+    jerry = os.path.join(options.jerry_location, 'build', 'bin', 'jerry')
+
+    print('\033[94m\nGenerating tests\n\033[0m')
+    for op in options.op:
```

Comment:
Since this line also quite similar to [this](https://github.com/szeged/js-test-generators/pull/2/files#diff-cdaef491dcf2848e25367299aad3c654R203). Please merge it.